src/api/bin/rails

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/ruby.ruby3.4
-APP_PATH = File.expand_path('../config/application', __dir__)
-require_relative '../config/boot'
-require 'rails/commands'
+#!/usr/bin/env ruby.ruby3.4
```

Comment:
Don't use /usr/bin/env rpmlint will complain about it...

Reply:
Fixed. This change is no longer included in any commit.